src/System.Data.Odbc/src/Common/System/Data/Common/ExternDll.Linux.cs

```diff
@@ -6,7 +6,7 @@ namespace System.Data.Common
 {
     internal static class ExternDll
     {
-        public const string Odbc32 = "odbc.so.2";
+        public const string Odbc32 = "libodbc.so.2";
```

Comment:
Why do these need renaming? How was it working before?

Reply:
When I was trying installation on the following machines:
- Debian 8.2, fedora 24, Ubuntu 16.04

When I do 
```> ldconfig -p | grep odbc```
I noticed that I get libodbc so that made me wonder if libodbc is the main dependency required. 

We could keep it as odbc rather than changing to libodbc since it is not failing already.

Reply:
If the library couldn't be found with the specified name, the loader tries variations on the name while looking for the library:
https://github.com/dotnet/coreclr/blob/46ab1d132c9ad471d79afa20c188c2f9c85e5f20/src/vm/dllimport.cpp#L6178-L6188
If we're able to provide the name as it's actually defined, that's best.

(But it should really follow our interop conventions and be defined as a const on Interop.Libraries.)

Reply:
I'm working on making this coherent with the interop conventions. Can I add that change in a separate PR? I'm afraid it's going to produce a large diff

Reply:
Yup, separate is great. Thanks.